src/app/tasks/tasks.module.ts

```diff
@@ -1,11 +1,13 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 
 import { TasksRoutingModule } from './tasks-routing.module';
 import { TasksComponent } from './tasks.component';
+import { MaterialModule } from '../material.module';
```

Comment:
please import the SharedModule. The SharedModule already imports the MaterialModule. 
Dependency graph between modules should be like:
![image](https://user-images.githubusercontent.com/15979292/104891391-8b6ca480-5979-11eb-9374-7aaad2bdf9d8.png)